clientconn.go

```diff
@@ -406,10 +408,11 @@ func getChainStreamer(interceptors []StreamClientInterceptor, curr int, finalStr
 // connectivityStateManager keeps the connectivity.State of ClientConn.
 // This struct will eventually be exported so the balancers can access it.
```

Comment:
Could you also please add a TODO here:
```golang
// TODO: If possible, get rid of the `connectivityStateManager` type, and 
// provide this functionality using the `Tracker`, to avoid keeping track of
// the connectivity state at two places.
```